bastionzero/service/connections/connections.go

```diff
@@ -93,6 +94,7 @@ type CreateUniversalConnectionRequest struct {
 	EnvironmentName string                `json:"envName,omitempty"`
 	TargetGroups    []string              `json:"targetGroups,omitempty"`
 	TargetType      targettype.TargetType `json:"targetType,omitempty"`
+	VerbType        verbtype.VerbType     `json:"verbType,omitempty"`
```

Comment:
`VerbType` is optional right? Just making sure

Reply:
Yes, do I need to change something?